File: src/game/gamego.c
#include <stdio.h>

/*
 
     Two Phase Game System
 
*/
extern FILE *openlock();

char **argv_p;
char privs[4];

 
listfl(name)
char *name;
{
FILE *a;
char b[128];
a=openlock(name,"r+");
while(fgets(b,128,a)) printf("%s\n",b);
fcloselock(a);
}
 
char *getkbd(s,l)   /* Getstr() with length limit and filter ctrl */
 char *s;
 int l;
    {
    char c,f,n;
    f=0;c=0;
    while(c<l)
       {
       regec:n=getchar();
       if ((n<' ')&&(n!='\n')) goto regec;
       if (n=='\n') {s[c]=0;f=1;c=l-1;}
       else
          s[c]=n;
       c++;
       }
    if (f==0) {s[c]=0;while(getchar()!='\n');}
    return(s);
    }

set_progname(n,text)
char *text;
{
	/*
	int x=0;
	int y=strlen(argv_p[n])+strlen(argv_p[1]);  
	y++;
	if(strcmp(argv_p[n],text)==0) return;
	
	while(x<y)
	   argv_p[n][x++]=0; 
	strcpy(argv_p[n],text);
	*/
}

File: src/stubs/gamego.py
"""
Two Phase Game System
"""
state = {
    'argv_p': '',
    'privs': [],
    'sig_active': 0,
    'interrupt': 0,
}


def main(argc, argv):
    raise NotImplementedError()


def crapup(__str):
    raise NotImplementedError()


def listfl(name):
    raise NotImplementedError()


def getkbd(s, l):
    raise NotImplementedError()


def sig_alon():
    raise NotImplementedError()


def unblock_alarm():
    raise NotImplementedError()


def block_alarm():
    raise NotImplementedError()


def sig_aloff():
    raise NotImplementedError()


def sig_occur():
    raise NotImplementedError()


def sig_init():
    raise NotImplementedError()


def sig_oops():
    raise NotImplementedError()


def sig_ctrlc():
    raise NotImplementedError()


def set_progname(n, text):
    raise NotImplementedError()
